spec/features/view_an_individual_list_spec.rb

```diff
@@ -19,7 +19,8 @@ def have_heading(title)
   end
 
   def have_book(title)
-    have_css("ul.books > li", text: title)
+    # TODO: rename to `.book` after #16 is merged
+    have_css(".booklist h1", text: title)
```

Comment:
Ok, looks good to use the default books style. I think you can now rename the specs and remove the comment above.